shaded-deps-dataproc20/pom.xml

```diff
@@ -19,4 +19,21 @@
         <tez.version>0.9.2</tez.version>
     </properties>
 
+    <build>
```

Comment:
I'm explicitly setting the plugins here (as well as in all the other children modules below) because it seems that Maven doesn't let children inherit the parent `plugins` clause -- only the `pluginsManagement` clause. Oddly, the `dependencies` clause does get inherited though, as you would expect.